README.md

```diff
@@ -86,7 +86,13 @@ vars:
 
 > 👀 Subscriptions and financial data are NOT included in `app_reporting` data models. This data is leveraged in the individual Google Play and Apple App Store packages, which are installed within the App Reporting package.
 
-## (Recommended) Step 5: Change the Build Schema
+## Step 5: Seed `country_codes` mapping tables (once)
+
+In order to map longform territory names to their ISO country codes, we have adapted the CSV from [lukes/ISO-3166-Countries-with-Regional-Codes](https://github.com/lukes/ISO-3166-Countries-with-Regional-Codes) to align Google and [Apple's](https://developer.apple.com/help/app-store-connect/reference/app-store-localizations/) country name formats for the App Reporting package. 
+
+You will need to `dbt seed` the `google_play__country_codes` [file](https://github.com/fivetran/dbt_google_play_source/blob/main/seeds/google_play__country_codes.csv) and `apple_store_country_codes` [file](https://github.com/fivetran/dbt_google_play_source/blob/main/seeds/apple_store_country_codes.csv) just once.
+
```

Comment:
```suggestion
https://github.com/fivetran/dbt_apple_store_source/blob/main/seeds/apple_store_country_codes.csv
```

Reply:
Think you meant to link the apple codes to this link, not google (404 error)

https://github.com/fivetran/dbt_apple_store_source/blob/main/seeds/apple_store_country_codes.csv